service/experiment-requests.yaml

```diff
@@ -16,6 +16,10 @@
 # You can run "make presubmit" to do basic validation on this file.
 # Please add new experiment requests towards the top of this file.
 
+- experiment: 2020-09-18
```

Comment:
Just fyi, had to change the date since 2020-09-17 is a taken name in db (we have a broken test experiment). Experiment will start soon regardless (i mean in a day since currently 2020-09-16 is using up all the cores).

Reply:
Yes, I would like to evaluate the newest version of aflsmart (a9d60257a6b5a7df2e177bddc6982376723bfd90) which support the new feature. Thanks.

Reply:
https://github.com/google/fuzzbench/blob/master/fuzzers/aflsmart/builder.Dockerfile#L41 has already been updated in my previous PR (#712)